users/tests/tests.py

```diff
@@ -217,35 +217,31 @@ def setUpTestData(cls):
         cls.made_out_in_spring_last_year = UsersHaveMadeOutFactory(
             user_one=cls.user,
             user_two=cls.user_two,
+            created=now.replace(year=now.year - 1, month=3, day=1)
         )
         cls.made_out_in_autumn_last_year = UsersHaveMadeOutFactory(
             user_one=cls.user,
             user_two=cls.user_two,
+            created=now.replace(year=now.year - 1, month=9, day=1)
```

Comment:
@alexaor When using TimeStampedModel, you can override the created/modified fields from the factory 😃